pkg/api/describe/describer.go

```diff
@@ -97,7 +98,8 @@ func describeBuilderImage(config *api.Config, image string, out io.Writer) {
 		Tag:                config.Tag,
 		IncrementalAuthentication: config.IncrementalAuthentication,
 	}
-	build.GenerateConfigFromLabels(c)
+	pr, _ := docker.GetBuilderImage(c)
```

Comment:
The error was ignored before.  Assumption I took was to describe as much as
possible.  I was inclined to not change that.

On Friday, August 12, 2016, Ben Parees <notifications@github.com> wrote:

> In pkg/api/describe/describer.go
> <https://github.com/openshift/source-to-image/pull/570#discussion_r74624778>
> :
>
> > @@ -97,7 +98,8 @@ func describeBuilderImage(config *api.Config, image string, out io.Writer) {
> >  		Tag:                config.Tag,
> >  		IncrementalAuthentication: config.IncrementalAuthentication,
> >  	}
> > -	build.GenerateConfigFromLabels(c)
> > +	pr, _ := docker.GetBuilderImage(c)
>
> why aren't you checking this error?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/source-to-image/pull/570/files/adcdd6aa20b77e9c9ec5dce3b524cf20b22042ca#r74624778>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADbadIK2fgbY9YEDd5Fg96RVaYb9zLaqks5qfKcIgaJpZM4JjLrA>
> .
>